ec2/sc-ec2-linux-jumpcloud.yaml

```diff
@@ -235,7 +235,6 @@ Resources:
         - !Ref 'InstanceRole'
   PrivateSubnetSecurityGroup:
     Type: 'AWS::EC2::SecurityGroup'
-    Condition: UsePrivateSubnet
```

Comment:
removing will always create this resource even if it's not being used?

Reply:
It will always be used, as with the templates in aws-infra

Reply:
actually all ports are open for private instances so you don't even need a PrivateSecurityGroup.  The security group in managed-ec2-linux-v1.j2 is only created and used when the user specifies OpenPorts and it only applies to public instances

Reply:
sorry i misunderstood that your PrivateSubnetSecurityGroup is actually just the same as the Vpn security group.  this looks good.